main/src/main/java/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1846,6 +1858,10 @@ public void setContent() {
             //register for changes of variableslist -> state of variable sync may change
             cache.getVariables().addChangeListener(this, s -> activity.adjustPersonalNoteVarsOutOfSyncButton(binding.personalnoteVarsOutOfSync));
 
+            final OfflineTranslateUtils.Status currentTranslationStatus = activity.translationStatus;
+            if (currentTranslationStatus.getSourceLanguage().isValid() && !currentTranslationStatus.isInProgress()) {
+                translateListing();
```

Comment:
setContent is called multiple times, so check for translation in progress to avoid concurring processes